CONTRIBUTING.md

```diff
@@ -79,85 +37,139 @@ confusing.  Here is a short guide on how to do it.
     (referred to as 'origin') as well as the main ManimCommunity repository
     (referred to as 'upstream').
 
-4. Choose the correct branch for your changes.  If you are working on a branch
-   that already exists on the main repository, you can execute
+4. Choose the branch for your changes.
+
+   If you want to work on a branch that already exists on the ManimCommunity repository
+   you can change to it with:
    ```sh
-   git checkout -b <branch-name> upstream/master
+   git checkout -b <branch-name> upstream/<branch-name>
```

Comment:
The original one was correct. People should be making an PR to master and not to any other branch,